src/main/resources/application.yml

```diff
@@ -20,4 +20,19 @@ swagger:
     name: Michael
     url: http://www.ebsco.com
     email: mpanson@ebsco.com
-  version: 1.0.0
\ No newline at end of file
+  version: 1.0.0
+  
+--- 
+spring:
+    profile: "non-test"
```

Comment:
I think you can leave this as "default" instead of creating a profile called "non-test". (In other words, don't add a profile at all)